src/components/map/MapPage.js

```diff
@@ -55,7 +55,8 @@ const MapPage = () => {
 
   useEffect(() => {
     async function fetchClusterAndLocationData() {
-      if (view.bounds) {
+      if (view.bounds?.ne.lng) {
```

Comment:
what does the '?' mean here? I assume ne.lng is north east longitude?

Reply:
Great question. `?.` is [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). I first learned about it here actually: https://2020.stateofjs.com/en-US/features/ (this is a really great visualization to explore!)